### PR TITLE
fix: Limit length of max invitees

### PR DESCRIPTION
**Pull request by sean-brydon**
Fixes invitee members length

### CODE REVIEW

File: packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts [hunk 1]
```diff
@@ -51,6 +53,13 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
 
   const emailsToInvite = await getEmailsToInvite(input.usernameOrEmail);
 
+  if (emailsToInvite.length > 25) {
```
Comment (PeerRich) [inline packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts:56]:
```suggestion
  if (emailsToInvite.length > 50) {
```

File: packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts [hunk 2]
```diff
@@ -51,6 +53,13 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
...
+    throw new TRPCError({
+      code: "PAYLOAD_TOO_LARGE",
+      message: "Please invite 25 members at a time",
```
Comment (PeerRich) [inline packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts:59]:
```suggestion
      message: "Please invite 50 members at a time",
```

Comment (PeerRich) [reply packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts:59]: lets do 50 instead

Comment (PeerRich) [reply packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts:59]: also, is there no better way for us to rate limit this? 🤔

**Review by PeerRich — APPROVED** [review]

File: packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts (hunk 2 above)
Comment (emrysal) [reply packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts:59]: IMO this is uglier than a global "You can only have 50 pending invites" - please poke your team members to send more.

with an exception of verified orgs

Comment (PeerRich) [reply packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts:59]: agree. lets just make invites paid. that will solve the spam issues. closing

**Comment by PeerRich** [comment]
lets just make invites paid. that will solve the spam issues. closing.

cc @zomars

**Pull request closed without merging**